spec/support/feature_helpers.rb

```diff
@@ -12,6 +12,7 @@ def login
   end
 
   def current_user
+    FactoryGirl.create(:country)
```

Comment:
Don't need to create the country here if it's set correctly in the user factory.